README.md

```diff
@@ -30,14 +30,124 @@ input {
 output {
  scalyr {
    api_write_token => 'SCALYR_API_KEY'
-   origin_field => 'host'
+   serverhost_field => 'host'
    logfile_field => 'path'
  }
 }
 ```
 
-In the above example, the Logstash pipeline defines a file input that reads from `/var/log/messages`.  Log events from this source have the `host` and `path` fields.  The pipeline then outputs to the scalyr plugin, which in this example is configured to remap `host`->`origin` and `path`->`logfile`, thus facilitating filtering in the Scalyr UI.
+In the above example, the Logstash pipeline defines a file input that reads from `/var/log/messages`.  Log events from this source have the `host` and `path` fields.  The pipeline then outputs to the scalyr plugin, which in this example is configured to remap `host`->`serverHost` and `path`->`logfile`, thus facilitating filtering in the Scalyr UI.
 
+## Options
+
+- The Scalyr API write token.  This is the only compulsory configuration field required for proper upload
+
+`config :api_write_token, :validate => :string, :required => true`
+
+---
+
+- If your Scalyr backend is located in other geographies (such as Europe), you may need to modify this
+
+`config :scalyr_server, :validate => :string, :default => "https://agent.scalyr.com/"`
+
+---
+
+- Path to SSL bundle file.
+
+`config :ssl_ca_bundle_path, :validate => :string, :default =>  "/etc/ssl/certs/ca-bundle.crt"`
+
+---
+
+- server_attributes is a dictionary of key value pairs that represents/identifies the logstash aggregator server
```

Comment:
Yes server_attributes have to represent the aggregator just because a single addEvents request could container multiple serverHosts. The option that adds the aggregator `serverHost` is off by default for this reason.

Reply:
We have the same situation with Kafka - a single task can be processing logs from multiple serverHosts.  Each serverHost / log file gets its own log id and there are multiple log ids in the `addEvents` `logs` array.  It seems like we should always have `serverHost` be the origin host of the log event and never the aggregator node.

Reply:
Unfortunately we can't guarantee that we will have a serverHost field in this setup, we can receive pretty arbitrary data in this plugin after all. The documentation does recommend adding these fields as we would like them but we can't actually enforce anything.